### PR TITLE
Fix bug in ipynb-render.sh

### PR DESCRIPTION
**Pull request by ethanwharris**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: sample-template/.meta.yml [hunk 1]
```diff
@@ -3,9 +3,11 @@ author: PL team
 created: 2021-06-15
 updated: 2021-06-17
 license: CC
-build: 3
+build: 4
 description: |
   This is a template to show how to contribute a tutorial.
+tags:
+  - Template
```
Comment (Borda) [inline sample-template/.meta.yml:10]: that is not probably relevant tag :]

**Comment by Borda** [comment]
@ethanwharris mind chek `WARNING: Title level inconsistent` and failing test for two notebooks

**Comment by Borda** [comment]
@ethanwharris can we make this land, seems be boking other UvA notebooks.. :rabbit: 
cc: @phlippe

**Comment by ethanwharris** [comment]
@Borda I've made a few fixes, hopefully the CI should pass now

**Review by Borda — APPROVED** [review]

**Pull request merged**